dataflow/flex-templates/streaming_beam/metadata.json

```diff
@@ -7,7 +7,7 @@
       "label": "Input PubSub subscription.",
       "helpText": "Name of the input PubSub subscription to consume from.",
       "regexes": [
-        "[/:-_.a-zA-Z0-9]+"
+        "[-_.a-zA-Z0-9]+"
```

Comment:
Some differences here:
1. The first character must be a letter (a-zA-Z)
2. It must be at least 3 characters long
3. We also support [-_.~+%]
4. It can't start with `goog`

Reply:
Thank you! I think I got it covered in the new commit :)